erpnext_documentation/www/docs/v13/user/manual/en/human-resources/grievance-type.md

```diff
@@ -0,0 +1,20 @@
+<!-- add-breadcrumbs -->
+# Grievance Type
+
+> Introduced in version-13
+
+**To resolve Employee grievances, first we need to classify them. For these grievances, the company may have to set different process for solving different Type of Grievance.**
```

Comment:
To resolve Employee grievances, we need to classify them. For this, the company may have to identify different Types of Grievances.

Reply:
Done